.circleci/config.yml

```diff
@@ -61,23 +78,43 @@ jobs:
       - run: *setup
       - run:
           name: Build Native Java App
-          command: ./test/test_force.js --os=android --apptype=native
+          command:  |
+            ./test/test_force.js --os=android --apptype=native
+            if [ ! -e tmp**/App_native_android/app/build/outputs/apk/debug/app-debug.apk ]; then
+              exit 1
+            fi
           when: always
       - run:
           name: Build Native Kotlin App
-          command: ./test/test_force.js --os=android --apptype=native_kotlin
+          command:  |
+            ./test/test_force.js --os=android --apptype=native_kotlin
+            if [ ! -e tmp**/App_native_kotlin_android/app/build/outputs/apk/debug/app-debug.apk ]; then
+              exit 1
+            fi
           when: always
       - run:
           name: Build Hybrid Local App
-          command: ./test/test_force.js --os=android --apptype=hybrid_local
+          command:  |
+            ./test/test_force.js --os=android --apptype=hybrid_local
+            if [ ! -e tmp**/App_hybrid_local_android/platforms/android/app/build/outputs/apk/debug/app-debug.apk ]; then
+              exit 1
+            fi
           when: always
       - run:
           name: Build Hybrid Remote App
-          command: ./test/test_force.js --os=android --apptype=hybrid_remote
+          command:  |
+            ./test/test_force.js --os=android --apptype=hybrid_remote
```

Comment:
So we can't work off the process exit code?

Reply:
I tried that for a bit with mixed results before realizing we shouldn't really care about the process exist code for a test script.  The point of the automation is to test generating and building the app.  So why not base pass/fail on whether we successfully built the .app or apk?

Reply:
That’s a good approach.